bundles/event_admin/event_admin_api/include/celix_event_admin_service.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef CELIX_EVENT_ADMIN_SERVICE_H
+#define CELIX_EVENT_ADMIN_SERVICE_H
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "celix_properties.h"
+#include "celix_errno.h"
+
+/**
+ * @brief The Event Admin service name
+ */
+#define CELIX_EVENT_ADMIN_SERVICE_NAME "org.osgi.service.event.EventAdmin"
```

Comment:
Although Apache Celix is inspired by OSGi and we did use some OSGi named constants, enums etc. 
I currently think it is better not to use the "org.osgi" or "osgi" part and replace that with "celix".

So in this case "celix.service.event.EventAdmin"